### DISCUSSION TITLE
Vs code doesnot show errors in detected by eslint in the file.

### DISCUSSION THREAD

**Discussion by sagarmsraozil** [eslint/eslint#18603] [Q&A]
The indicators for errors was working fine in the vs code. but suddenlty its not working. I dont know why. I have tried everything found on the web. But its not working. Please help.

 "eslint.execArgv": null,
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  },
  "eslint.validate": [
    "javascript",
    "typescript",
    "javascriptreact",
    "typescriptreact"
  ],
  
  i have already added it

**Comment by harish-sethuraman** [comment]
Can you once try `>eslint: restart eslint server` command inside vs code by doing `cmd + shift + p`.
If on mac or `ctrl + shift + p` if on windows?

**Reply by sagarmsraozil** [reply]
@harish-sethuraman didnot work with that.

The error i get is

Command eslint:restart eslint server resulted in an error
command 'eslint.restart' not found.
